alternator/stats.cc

```diff
@@ -156,6 +156,8 @@ static void register_metrics_with_optional_table(seastar::metrics::metric_groups
                     [&stats]{ return estimated_histogram_to_metrics(stats.api_operations.batch_write_item_histogram);})(op("BatchWriteItem")).aggregate({seastar::metrics::shard_label}).set_skip_when_empty(),
             seastar::metrics::make_histogram("operation_size_kib", seastar::metrics::description("Histogram of item sizes involved in a request"), labels,
                     [&stats]{ return estimated_histogram_to_metrics(stats.operation_sizes.get_item_op_size_kib);})(op("GetItem")).aggregate({seastar::metrics::shard_label}).set_skip_when_empty(),
+            seastar::metrics::make_histogram("operation_size_kib", seastar::metrics::description("Histogram of item sizes involved in a request"), labels,
+                    [&stats]{ return estimated_histogram_to_metrics(stats.operation_sizes.batch_get_item_op_size_kib);})(op("BatchGetItem")).aggregate({seastar::metrics::shard_label}).set_skip_when_empty(),
```

Comment:
I think there's a typo in the commit message, which refers to BatchWriteItem in some places, instead of BatchGetItem.